pkg/api/instances.go

```diff
@@ -260,6 +268,22 @@ func (api *API) instanceAppQuery(appID string) *dat.SelectDocBuilder {
 func ignoreFakeInstanceCondition(instanceIDField string) string {
 	return fmt.Sprintf(`(%[1]s IS NULL OR %[1]s NOT SIMILAR TO '\{[a-fA-F0-9-]{36}\}')`, instanceIDField)
 }
+func (api *API) getFilterInstanceCount(p InstancesQueryParams) *dat.SelectBuilder {
+	instancesSubquery := api.dbR.
+		Select("COUNT (*)").
+		From("instance_application").
+		Where("application_id = $1 AND group_id = $2", p.ApplicationID, p.GroupID).
+		Where(fmt.Sprintf("last_check_for_updates > now() at time zone 'utc' - interval '%s'", validityInterval)).
```

Comment:
```suggestion
		Where("last_check_for_updates > now() at time zone 'utc' - interval $1", validityInterval).
```

Reply:
@ashu8912 You should accept Krzesimir's suggestion and squash it with your commit.

Reply:
Could you fix this one too? Thanks.
```suggestion
		Where("last_check_for_updates > now() at time zone 'utc' - interval $1", validityInterval).
```